### ISSUE TITLE
Set up a domain tutorial make cluster-upgrade docker dependency

### ISSUE THREAD

**Issue by DougForrest** [cvisionai/tator#443]
Following the tutorial[ Set up a domain ](https://www.tator.io/tutorials/2021-03-30-set-up-a-domain/) which is a suggested follow up to [Install tator](https://www.tator.io/tutorials/2021-03-29-install-tator/) requires docker which is not installed in install.sh and it is unnecessary to rebuild images at this step.

`Makefile cluster-upgrade` depends on the images target. 
```bash
cluster-upgrade: check-migration main/version.py images
	helm upgrade --debug --atomic --timeout 60m0s --set gitRevision=$(GIT_VERSION) tator helm/tator
```

images target
```bash
.PHONY: images
images:
	make ${IMAGES}
```

images variable all built with docker
```bash
IMAGES=python-bindings postgis-image client-image
```

As only one value is changed in values.yaml running helm upgrade should be sufficient or removing the image dependency as shown below is also sufficient. 

```bash
cluster-upgrade: check-migration main/version.py
	helm upgrade --debug --atomic --timeout 60m0s --set gitRevision=$(GIT_VERSION) tator helm/tator
```

Taking a step back it would be nice if install.sh allowed for an external IP / domain variable which would negate the need for the setup a domain step. As far as I can tell the only place this variable needs to be set is the Django settings.py `ALLOWED_HOSTS`

Thanks again for helping me through the setup.

**Comment by jrtcppv** [comment]
This is a good point that I think could be addressed with a new `make cluster-config` target that would drop the images dependency. Then `cluster-upgrade` could remain a target used by developers only. It is true that we often only need to apply changes to `values.yaml` without building the images.

The tutorial may not be clear on this, but I believe you could set the `HOST_IP` environment variable to a domain rather than an IP address and as long as the domain is pointed to the correct IP address you can accomplish the domain setup in the install step. I will test this out and update the tutorial if it works.

**Comment by jrtcppv** [comment]
I pushed out a new command `cluster-update` that only does the helm upgrade. I tested it on a fresh install and it worked fine. I will roll this into the next release and update the domain tutorial (and elsewhere that we are using `cluster-upgrade` if it isn't necessary).

I also tried setting `HOST_IP` to a domain in a fresh install and I forgot that the IP address is used by the install script for both MetalLB's IP range and the Django allowed hosts that you mentioned. It would be possible to add a separate environment variable for domain, but I want to keep `install.sh` as close to zero config as possible and also keep that first install tutorial as short as possible -- we used to roll up all the config options in one giant tutorial and it was intimidating to say the least. Starting with just IP address also mitigates the potential for DNS errors associated with using a domain at the outset.

Thanks again for helping us smooth out the rough edges on install!
